words/predicates.txt

```diff
@@ -235,15 +276,17 @@ delicate
 delicious
 delightful
 delirious
+deluxe
 denim
 dent
+dented
 descriptive
 desert
 deserted
 destiny
 detailed
 determined
-dew
+dewy
```

Comment:
umm....

Reply:
maybe we should yank this one

Reply:
is dew a predicate? I get dewy being awkward, this project stretches the limits of my part-of-speech knowledge.

Reply:
The only word I can imagine following dew is 'drop'. Dewy is more of a predicate but also more sensual sounding, so I second yanking it.